core/src/apps/webauthn/__init__.py

```diff
@@ -814,7 +814,7 @@ def __init__(self, cid: int, iface: io.HID, rp_id: str) -> None:
         text.normal(
             "This device is not", "registered with", self._creds[0].app_name() + "."
         )
-        await confirm(text, confirm=None)
+        await confirm(text, confirm=None, cancel="Close")
```

Comment:
Before and after:

<img src="https://user-images.githubusercontent.com/42678794/65409441-05256a00-dde8-11e9-92b6-76ea8b52c2e0.png" width=300><img src="https://user-images.githubusercontent.com/42678794/65409447-0787c400-dde8-11e9-84ec-14dee522f3f7.png" width=300>